docs/src/appendix/snfoundry-toml.md

```diff
@@ -0,0 +1,101 @@
+# The Manifest Format
+
+The `snfoundry.toml` file is not included in every package by default. If you want to use it, you need to create it yourself. 
```

Comment:
Since it's created automatically [here](https://github.com/foundry-rs/starknet-foundry/blob/2a1be2f27d06f15b56246a3d0fa3952306422d46/crates/forge/src/init.rs#L140), it's worth mentioning that newly created projects are going to have it.